crates/dbs-virtio-devices/src/device.rs

```diff
@@ -68,8 +68,9 @@ impl<Q: QueueStateT> VirtioQueueConfig<Q> {
     pub fn create(queue_size: u16, index: u16) -> Result<Self> {
         let eventfd = EventFd::new(EFD_NONBLOCK).map_err(Error::IOError)?;
 
+        let queue = Q::new(queue_size)?;
```

Comment:
This part need to be reviewed more carefully.
Other parts are just changeing the parameter names.